crawl-ref/source/dat/strings/de/items-unique.txt

```diff
@@ -415,7 +415,7 @@ an opalescent scaly cloak
 %%%%
 # sword of Cerebov
 a flamberge
-{m}ein Flamberg
+{m}ein Flamberge
```

Comment:
Duden and DWDS both say Flamberg
https://www.duden.de/rechtschreibung/Flamberg
https://www.dwds.de/wb/Flamberg

Reply:
Oops. You're right again. Flamberge is sometimes used too, but Flamberg is much more common.